vuepress/docs/next/docs/compose/ecm-overview.md

```diff
@@ -0,0 +1,20 @@
+# Entando Component Manager
+
+​​An Entando Application is composed of the Entando App Builder, Entando App Engine, and Entando Component Manager. The Entando Component Manager (ECM) is a service that links the [Entando Component Repository](ecr-overview.md) (ECR) of the App Builder to the core application instance. It appears as `quickstart-cm-deployment` in the Kubernetes pod list:
+
+![pods.png](./img/pods.png)
+
+The ECM communicates with the Entando Kubernetes service to populate the ECR with the bundles deployed as [Custom Resources](../consume/custom-resources.md) in the Entando cluster. These bundles are available to the application and can be managed from within the ECR. 
```

Comment:
minor point, in the first line, 'Entando Kubernetes service' >> can we drop the Entando? The first paragraph has so many Entando's, which was necessary, wondering if it's absolutely needed here?

Reply:
i'm under the impression that this kubernetes service has been modified or integrated to be entando-specific. if that's the case, it *might* be important to note. i left it in to err on the side of caution, but i agree that we should drop the "Entando" if possible. @nshaw?

Reply:
removed "Entando"